Assets/css/style.css

```diff
@@ -0,0 +1,33 @@
+/* Navigation Bar Styles */
+ul {
```

Comment:
Please replace "ul" with "nav ul" just so it doesnt effect all UL's on the page

Reply:
okay